task/03-date-tasks.js

```diff
@@ -76,14 +77,29 @@ function isLeapYear(date) {
  *    Date(2000,1,1,10,0,0),  Date(2000,1,1,15,20,10,453)   => "05:20:10.453"
  */
 function timeSpanToString(startDate, endDate) {
-   throw new Error('Not implemented');
+  let duration = endDate-startDate
+  let hours = Math.floor(duration / (3600*1000))
+  if(hours<10) hours = '0' + hours
+  let minutes = Math.floor((duration - (hours*3600*1000))/(60*1000))
+  if(minutes<10) minutes = '0' + minutes
+  let seconds = Math.floor((duration - (hours*3600*1000)-(minutes*60*1000))/1000)
+  if(seconds<10) seconds = '0' + seconds
+  let milliseconds = duration - (hours*3600*1000) - (minutes*60*1000) - (seconds*1000)
+  if( milliseconds<100)  milliseconds = '0' +  milliseconds
```

Comment:
It is necessary to follow the general style of the code: 
- indentation after the keywords "for", "if", etc.
- spaces between operators "*", "<", etc.
- remove extra spaces "return`  $ {hours}: " and "if ( milliseconds", and so on.
Also, comments that are not related to the current code are not needed.